cmd/storagenode/dashboard.go

```diff
@@ -19,12 +19,13 @@ import (
 	"go.uber.org/zap"
 
 	"storj.io/storj/internal/memory"
-	"storj.io/storj/pkg/identity"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/process"
 	"storj.io/storj/pkg/transport"
 )
 
+const contactWindow = time.Hour * 4
```

Comment:
`contactWindow` looks to be quite long but I have no strong opinion about that. @stefanbenten or @littleskunk

Reply:
nevermind, its just about colors in dashboard.